pages/common/find.md

```diff
@@ -25,3 +25,11 @@
 - delete files by name, older than a certain number of days
 
 `find {{root_path}} -name {{'*.py'}} -mtime {{-180d}} -delete`
+
+- find empty files
+
+`find -empty`
```

Comment:
Looks like the MacOS version of find requires explicitly a path to be specified (http://superuser.com/questions/529186/how-can-i-delete-empty-folders-in-mac-os-x), while in the GNU version it can be ommited. I'll have this fixed.